plugins/inputs/webhooks/plex/plex_webhooks.go

```diff
@@ -0,0 +1,43 @@
+package plex
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/influxdata/telegraf"
+)
+
+type PlexWebhook struct {
+	Path string
+	acc  telegraf.Accumulator
+}
+
+func (p *PlexWebhook) Register(router *mux.Router, acc telegraf.Accumulator) {
+	router.HandleFunc(p.Path, p.eventHandler).Methods("POST")
+	log.Printf("I! Started the webhooks_plex on %s\n", p.Path)
+	p.acc = acc
+}
+
+func (p *PlexWebhook) eventHandler(w http.ResponseWriter, r *http.Request) {
+	r.ParseMultipartForm(1 << 20)
+	e, err := generateEvent([]byte(r.Form["payload"][0]), &PlexWebhookEvent{})
```

Comment:
This is also causing a panic in the tests. Verify the length of `r.Form` before getting here.